runtests.py

```diff
@@ -67,6 +67,8 @@ def main(argv):
     parser = ArgumentParser(usage=__doc__.lstrip())
     parser.add_argument("--verbose", "-v", action="count", default=1,
                         help="more verbosity")
+    parser.add_argument("--debug-configure", action="store_true",
+                        help="add -v to build_src to show cconfiguration compiler output")
```

Comment:
This is pretty vague; can we describe what the actual feature is that is enabled?

Reply:
"adds -v to the build_src subcommand which will show compiler output while probing platform features used to create _numpyconfig.h and config.h" seems a bit too much. 

Any ideas what would be more appropriate?

Reply:
I think the problem is that there are several things being addressed here: `-v` is being added to `build_src`, but only for a certain type of build?  Or is it always added?  If always, then the latter part of the sentence is not necessary.  It can simply be something like "Increase verbosity of build_src by adding -v flag".  But the "configure" in `--debug-configure` makes me think it was intended for something more specific.

Reply:
another option would be to not add it to runtests.py at all (which would always build without the flag), and in CI explicitly do `setup.py build build_src -v; runtests.py`.

Reply:
> and in CI explicitly do `setup.py build build_src -v; runtests.py`

That doesn't seem desirable at all, we want to run CI in the ways we expect humans to use things, not some arcane invocation. There's no real issue here, so why change anything?

Reply:
Yes, I think just a slightly more descriptive help string would be sufficient.